packages/react/src/components/Modal/Modal.types.ts

```diff
@@ -181,10 +181,11 @@ export interface IModalProps extends React.RefAttributes<HTMLDivElement>, IAcces
   allowTouchBodyScroll?: boolean;
 
   /**
-   * Puts aria-hidden=true on all non-ancestors of the current modal, for screen readers.
+   * Puts aria-hidden=true on all non-ancestors of the current modal, for screen readers
+   * (unless `isModeless` is true).
    * @defaultvalue true
    * @deprecated Setting this to `false` is deprecated since it breaks modal behavior for some screen readers.
-   * It will not be supported in future versions of the library.
+   * It will not be supported in future versions of the library (except for modeless modals).
```

Comment:
Do we want to specify this? I don't know if we'll support this specific prop in future versions (or at least I hope not) 😬

Reply:
The idea was that the behavior will still be the default for modeless modals (assuming those are supported), even though the prop will be gone. There's probably a better way to say it.